public/js/components/ManagedEditor/ManagedEditorForm.js

```diff
@@ -23,12 +25,23 @@ export class ManagedForm extends React.Component {
     });
   }
 
+  hasFormErrors(formErrorsObject) {
+    const fieldWithError = Object.keys(formErrorsObject).find((key) => formErrorsObject[key].length > 0);
+    return !!fieldWithError;
+  }
+
   updateFormErrors = (fieldErrors, fieldName) => {
+
+    const formErrors = Object.assign({}, this.state.formErrors, {
+      [fieldName]: fieldErrors
+    });
+
     this.setState({
-      formErrors: {
-        [fieldName]: fieldErrors
-      }
+      formErrors: formErrors
     });
+
+    this.props.onFormErrorsUpdate && this.props.onFormErrorsUpdate(formErrors);
```

Comment:
The parent can pass down a function that is called with the latest errors (similar to an onUpdate) this checks if it's been passed and calls it with the latest errors